examples/decoding/plot_decoding_time_generalization_conditions.py

```diff
@@ -25,10 +25,14 @@
 # License: BSD (3-clause)
 
 import numpy as np
+import matplotlib.pyplot as plt
 
 import mne
 from mne.datasets import sample
-from mne.decoding import GeneralizationAcrossTime
```

Comment:
Yes, the example has moved to the tutorial section at some point.

On 24 July 2016 at 14:48, Alexandre Gramfort <notifications@github.com>
wrote:

> In examples/decoding/plot_decoding_time_generalization_conditions.py
> <https://github.com/mne-tools/mne-python/pull/3381#discussion_r71993256>:
>
> >
> >  import mne
> >  from mne.datasets import sample
> > -from mne.decoding import GeneralizationAcrossTime
>
> GeneralizationAcrossTime is not used in any example any more? Maybe in
> tutorials?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3381/files/8be49243aa6833c9039981ed4bb4b429f9ce934c#r71993256>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEp7DBeLtJcQ7mP1CD7gGfPx36sm7KoLks5qY7NxgaJpZM4JDsU1>
> .
>